build.sh

```diff
@@ -34,18 +59,15 @@ case $ROS_DISTRO in
     ;;
 esac
 
-# make output directory
-mkdir /home/runner/apt_repo
-
 echo "Add unreleased packages to rosdep"
 
 for PKG in $(catkin_topological_order --only-names); do
-  printf "%s:\n  %s:\n  - %s\n" "$PKG" "$DISTRIBUTION" "ros-$ROS_DEB$(printf '%s' "$PKG" | tr '_' '-')" >> /home/runner/apt_repo/local.yaml
+  printf "%s:\n  %s:\n  - %s\n" "$PKG" "$DISTRIBUTION" "ros-$ROS_DEB$(printf '%s' "$PKG" | tr '_' '-')" >> ${APT_REPO}/local.yaml
```

Comment:
dito this.

btw. please don't copy parts of a commit from a different author out of context without proper attribution (use `git commit --author`, maybe)

Reply:
Didn't mean to steal your attribution at all, sorry. `git commit --author` is fine and I can add you in the commit message if that's what do you prefer.